README.md

```diff
@@ -63,6 +63,20 @@ documents all of these.  The most commonly used flags:
 * `-Dmodernizer.failOnViolations` - fail phase if violations detected, defaults to true
 * `-Dmodernizer.skip` - skip plugin execution, defaults to false
 
+Suppression
+-----------
+To suppress modernizer errors, add the following to the `<dependencies>` stanza in your pom.xml
```

Comment:
`stanza` -> `section`; this is more standard terminology

Reply:
Oh. I was just following the existing terminology in the file. `stanza` is the phrase used everywhere else - was trying to be consistent with it.

Reply:
Oh, I didn't realize that. Good catch 😄